File: internal/content/testdata/bench/csharp/sieve-of-eratosthenes.cs
public class Solution {
    public long[] primes_up_to_n(long n) {
        if (n < 2) return new long[0];
        bool[] isPrime = new bool[n + 1];
        for (long i = 2; i <= n; i++) isPrime[i] = true;
        for (long i = 2; i * i <= n; i++) {
            if (isPrime[i]) {
                for (long j = i * i; j <= n; j += i) isPrime[j] = false;
            }
        }
        var result = new List<long>();
        for (long i = 2; i <= n; i++) {
            if (isPrime[i]) result.Add(i);
        }
        return result.ToArray();
    }
}

File: internal/content/testdata/bench/go/sieve-of-eratosthenes.go
func primes_up_to_n(n int) []int {
	if n < 2 {
		return []int{}
	}
	isPrime := make([]bool, n+1)
	for i := 2; i <= n; i++ {
		isPrime[i] = true
	}
	for i := 2; i*i <= n; i++ {
		if isPrime[i] {
			for j := i * i; j <= n; j += i {
				isPrime[j] = false
			}
		}
	}
	result := []int{}
	for i := 2; i <= n; i++ {
		if isPrime[i] {
			result = append(result, i)
		}
	}
	return result
}
